docs/nerf_dataset_tips.md

```diff
@@ -129,6 +131,35 @@ With an >=iPhone 12 Pro, one can use [Record3D](https://record3d.app/) to collec
 	```
 	instant-ngp$ ./instant-ngp path/to/data
 	```
+	
+### NeRFCapture
+[NeRFCapture](https://github.com/jc211/NeRFCapture) is an iOS app that runs on any ARKit device. It allows you to stream images directly from your phone to InstantNGP thus enabling a more interactive experience. It can also collect an offline dataset for later use. 
+
+The following dependencies are needed to run the NeRFCapture script:
+```
+pip install cyclonedds
+pip install opencv-python
```

Comment:
__instant-ngp__ already contains `opencv-python-headless` in its requirements. Is that not enough? The non-headless version can be troublesome on some systems due to GUI dependencies.